README.md

```diff
@@ -1182,7 +1186,226 @@ Messages sent have a priority based on their Service Type. Lower values for serv
 ### 5.1 Control Service
 >Required: All Protocol Versions
 
-The control service is the lowest level service available. While Control Frame packets are used frequently, the control service itself is rarely used.   
+The control service is the lowest level service available. While Control Frame packets are used frequently, the control service itself is rarely used.
+
+#### 5.1.1 Security Query
+
+The TLS Payload is sent in a binary query header. The size is 12 bytes which matches the size of the RPC Payload Binary Header.
+
+##### 5.1.1.1 Payload
+
+The security query is able to contain JSON data as well as binary data. During the handshake the TLS handshake data is sent as binary data. In case of an error, a notification is sent with an error code and error message as JSON data. See "Error handling" section for details.
+
+<table width="100%">
+  <tr><td align="center">Binary Query Header</td></tr>
+  <tr><td align="center">JSON Data</td></tr>
+  <tr><td align="center">Binary Data</td></tr>
+</table>
+
+##### 5.1.1.2 Binary Header
+
+<table width="100%">
+  <tr>
+    <th width="25%">Byte 1</th>
+    <th width="25%">Byte 2</th>
+    <th width="25%">Byte 3</th>
+    <th width="25%">Byte 4</th>
+  </tr>
+  <tr>
+    <td>Query Type</td>
+    <td colspan="3" align="center">Query ID</td>
+  </tr>
+  <tr>
+    <td colspan="4" align="center">Sequential Number</td>
+  </tr>
+  <tr>
+    <td colspan="4" align="center">JSON Size</td>
+  </tr>
+</table>
+
+###### 5.1.1.2.1 Binary Header Fields
+
+<table width="100%">
+  <tr>
+    <th>Field</th>
+    <th>Size</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>Query Type</td>
+    <td>8 bit</td>
+    <td>
+      0x00 Request <br>
+      0x01 - 0x0F Reserved<br>
+      0x10 Response<br>
+      0x11 - 0x1F Reserved<br>
+      0x20 Notification<br>
+      0x21 - 0xFE Reserved<br>
+      0xFF Invalid Query Type
+    </td>
+  </tr>
+  <tr>
+    <td>Query ID</td>
+    <td>24 bit</td>
+    <td>
+      0x000001 Send Handshake Data<br>
+      0x000002 Send Internal Error<br>
+      0x000003 - 0xFFFFFE Reserved<br>
+      0xFFFFFF Invalid Query ID
+    </td>
+  </tr>
+  <tr>
+    <td>Sequential Number</td>
+    <td>32 bit</td>
+    <td>
+      Message ID can be set by the Mobile libraries to track security messages.
+      The system uses the same Message ID when replying to the query allowing the mobile to correlate messages.
+    </td>
+  </tr>
+  <tr>
+    <td>JSON Size</td>
+    <td>32 bit</td>
+    <td>The size of the JSON data following the binary query header.</td>
+  </tr>
+</table>
+
+#### 5.1.2 Error Frames
```

Comment:
```suggestion
#### 5.1.1.2 Send Internal Error
```
This should probably be a subsection of "Security Query", as should `Send Handshake Data`

Reply:
@jacobkeeler done. Now both "Send Handshake Data" and "Send Internal Error" are specified as separate subsections. https://github.com/smartdevicelink/protocol_spec/pull/41/commits/14c425acd8be165cc36d355bfb658c98b336240a